test/fixes/typedefs.unit

```diff
@@ -0,0 +1,49 @@
+40 columns                              |
+>>> simple typedef
+typedef int Foo(int x);
+<<<
+typedef Foo = int Function(int x);
+>>> generic typedef
+typedef S Foo<S extends num>(S x);
+<<<
+typedef Foo<S extends num> = S Function(
+    S x);
+>>> named argument
```

Comment:
This doesn't look like a named argument to me. :)

Reply:
done